README.md

```diff
@@ -17,16 +17,32 @@ Then all you have to do is clone this repo and invoke vagrant:
     cd /vagrant
     FLASK_APP=service:app flask run -h 0.0.0.0
     
-You must pass the parameters -h 0.0.0.0 to have it listed on all network adapters to that the post can be forwarded by vagrant to your host computer so that you can open the web page in a local browser at: http://localhost:5000/wishlists
-
 ### In case you already have a Vagrant VM
     vagrant up
     vagrant provision
     vagrant ssh
     cd /vagrant
     FLASK_APP=service:app flask run -h 0.0.0.0
-
-## Manually running the Tests
+    
+## Features supported
+
+ GET http://localhost:5000/wishlists - Return all the wishlists 
+ 
+ GET http://localhost:5000/wishlists/{wishlistID} - Return the wishlists with the given id  
+ 
+ POST http://localhost:5000/wishlists - Create a new wishlist record in the database  
+
+ PUT http://localhost:5000/wishlists/{wishlistID} - Update a wishlist record in the database  
+ 
+ DELETE http://localhost:5000/wishlists/{wishlistID} - Delete a wishlist record in the database  
+ 
+ GET http://localhost:5000/wishlists?name="wishlist_1" - Query the database by the name of the wishlist   
+
+ PUT http://localhost:5000/wishlists/{wishlistId}/disabled - It disables the target wishlist
+ 
+ PUT http://localhost:5000/wishlists/{wishlistId}/enabled - It enables the target wishlist
+ 
+ ## Manually running the Tests
```

Comment:
This is nice. Thank you for adding this. We have features for CRUD operations on items as well.
/wishlists/<int:wishlist_id>/items
/wishlists/<int:wishlist_id>/items/<int:item_id> etc
Let's add them too!

Reply:
Add create and get items in Read.md. It seems we don't have list and delete now. Please check again.

Reply:
We can add them when #10 is raised.